<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriviaGame</class>
 <widget class="QWidget" name="TriviaGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="answerButtonA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>480</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210);
border: 0;
padding: 10px;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="contentA">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border: 0;
padding: 10px 10px 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="contentB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border: 0;
padding: 10px 10px 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="contentC">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border: 0;
padding: 10px 10px 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="contentD">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border: 0;
padding: 10px 10px 10px 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelA</zorder>
  <zorder>labelD</zorder>
  <zorder>labelB</zorder>
  <zorder>labelC</zorder>
  <zorder>questionLabel</zorder>
  <zorder>contentA</zorder>
  <zorder>contentB</zorder>
  <zorder>contentC</zorder>
  <zorder>contentD</zorder>
  <zorder>answerButtonA</zorder>
  <zorder>answerButtonB</zorder>
  <zorder>answerButtonC</zorder>
  <zorder>answerButtonD</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
